Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml

```diff
@@ -22,6 +22,7 @@
   - type: StorageFill
     contents:
     - id: SmokingPipeFilledTobacco
+    - id: ZippoLighter
   - type: Armor #same as regular sec armor
```

Comment:
Mixing changes into a refactor is a bad idea and these should be separated out as it makes it a pain for forks to pull the changes they want without manually overwriting them, same with the other yaml changes outside of bugfixes.

Reply:
Yeah... I knew this would be a bad idea. I'll remove the Zippo from this PR and make another one once this one gets merged.

Reply:
Guess I should remove the e-sword hum and swing noises too, right?

Reply:
I'll leave those in as part of the sound fix for the e-sword.